test/functional/page_objects/dashboard_page.ts

```diff
@@ -468,14 +469,21 @@ export class DashboardPageObject extends FtrService {
       await this.testSubjects.existOrFail('saveDashboardSuccess');
     });
     const message = await this.common.closeToast();
-    await this.header.waitUntilLoadingHasFinished();
+
+    if(!skipLoadingIndicatorHiddenCheck) {
+      await this.header.waitUntilLoadingHasFinished();
```

Comment:
This line throws timeout error in serverless test suites. So added an option boolean flag to skip waiting when in serverless.